### ISSUE TITLE
External links should have an icon.

### ISSUE THREAD

**Issue by lavisht22** [topcoder-archive/beta-testing-dashboard#53]
Accessibility: All external links on the page should have a icon (indicator) to tell people that this will open a new tab or web page.

![image](https://user-images.githubusercontent.com/22209892/133471661-9103bf3c-2e8f-49b1-94a4-ba415233c30e.png)

**Comment by lavisht22** [comment]
![image](https://user-images.githubusercontent.com/22209892/133471828-538e0d6e-5568-4782-a323-d658c6805948.png)

**Comment by lavisht22** [comment]
![image](https://user-images.githubusercontent.com/22209892/133471885-4abd1df0-8da8-4f9d-911b-c12bc4c2e6a9.png)
